README.md

```diff
@@ -2,36 +2,37 @@
 
 ![Deploy and release](https://github.com/asreview/asreview-visualization/workflows/Deploy%20and%20release/badge.svg)![Build status](https://github.com/asreview/asreview-visualization/workflows/test-suite/badge.svg)
 
-This is a plotting/visualization supplemental package for the 
-[ASReview](https://github.com/asreview/asreview)
-software. It is a fast way to create a visual impression of the ASReview with different
-dataset, models and model parameters.
+This is a plotting/visualization supplemental package for the
+[ASReview](https://github.com/asreview/asreview) software. It is a fast way to
+create a visual impression of the ASReview with different datasets, models and
+model parameters.
 
 ## Installation
 
-The easiest way to install the visualization package is to use the command line:
+The easiest way to install the visualization package is to install from PyPI:
 
 ``` bash
 pip install asreview-visualization
 ```
 
-After installation of the visualization package, asreview should automatically detect it.
-Test this by:
+After installation of the visualization package, `asreview` should automatically
+detect it. Test this by:
 
 ```bash
 asreview --help
 ```
 
-It should list the 'plot' modus.
+It should list the 'plot' extension.
```

Comment:
Well, I would say that "plot" is a modus, while asreview-visualization is an extension (with possibly more than a single modus).